teos/src/gatekeeper.rs

```diff
@@ -213,13 +204,16 @@ impl Gatekeeper {
     pub(crate) fn add_update_appointment(
         &self,
         user_id: UserId,
-        uuid: UUID,
+        locator: Locator,
         appointment: &ExtendedAppointment,
     ) -> Result<u32, NotEnoughSlots> {
         // For updates, the difference between the existing appointment size and the update is computed.
         let mut registered_users = self.registered_users.lock().unwrap();
         let user_info = registered_users.get_mut(&user_id).unwrap();
-        let used_slots = user_info.appointments.get(&uuid).map_or(0, |x| *x);
+        // can be fetched from the DB, or combined in the update_user call.
+        // NOTE: this method's interaction with the DB needs to be combined with the addition of the appointment
+        // to make the process atomic.
+        let used_slots = 0;
```

Comment:
Yeah setting this as a dummy number for now.
Updates happen actually to be some pain in the butt :joy:
Could be fixed with a DB call to check if this appointment already exists and how many slots does it take, but I was trying to avoid doing DB here + I will need to update this method to actually do everything in one DB tx (compute the slot difference + update the userinfo + add the appointment).